lib/src/cardano_version.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/foundation.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'cardano_version.freezed.dart';
+
+@freezed
+class CardanoVersion with _$CardanoVersion {
+  const CardanoVersion._();
+
+  const factory CardanoVersion({
+    required bool testMode,
+    required int versionMajor,
+    required int versionMinor,
+    required int versionPatch,
+    required bool locked,
+  }) = _CardanoVersion;
+
+  /// Get the version code.
+  int get versionCode =>
```

Comment:
```dart
int get versionCode =>
      versionMajor * 10000 + versionMinor * 100 + versionPatch;
```

can be changed to 
```dart
late final int versionCode =
      versionMajor * 10000 + versionMinor * 100 + versionPatch;
```

This would avoid calculating the version code every time.

Reply:
done